services/ymax-planner/src/plan-deposit.ts

```
@@ -76,25 +66,40 @@ export const handleDeposit = async (
   spectrum: SpectrumClient,
   cosmosRest: CosmosRestClient,
 ) => {
-  const currentBalances = await getCurrentBalances(
-    portfolioKey,
-    readPublished,
-    spectrum,
-    cosmosRest,
-    amount.brand,
+  const powers = { spectrum, cosmosRest };
+  const querier = makePortfolioQuery(readPublished, portfolioKey);
+  const status = await querier.getPortfolioStatus();
+  const { targetAllocation, positionKeys, accountIdByChain } = status;
+  if (!targetAllocation) return;
```

Consider throwing/logging with context instead of silent return.

---

AIUI, it is acceptable and expected for a user to deposit funds before defining a target allocation.